modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,25 +1,30 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+Perhaps your organization requires someone with the proper authority to deploy Mule applications once the design phase has been completed.
+Flow Designer allows you to export the project as a `.jar` file and share it with those who are responsible for Mule app deployment.
+You'll need to use *Export mule application*, as it produces a `.jar` file that includes all the connector credentials and artifacts required by the Mule runtime.
 
-== Procedure
-
-. In Flow Designer, follow these steps:
-.. Ensure that your project is free of errors.
-.. Click the download icon, which is to the right of the Undo and Redo arrows in the top-left corner of the screen.
-.. Select *Download Package*.
+. In Flow Designer, ensure that your project is free of errors.
+. Export your project by clicking the dots next to the Test button and select *Export* from the list.
+. Select *Export mule application* for the entire app, and click *Export*.
 +
-*Result:* A `.jar` file that contains a copy of your project is downloaded to the default location on your computer.
+A `.jar` file containing a copy of your Mule app downloads to the default location on your computer.
+
 . Place the `.jar` file in a location that the other person can access.
 . Have the person receiving the `.jar` file follow these steps in Runtime Manager:
```

Comment:
I'm trying to think of a way that avoids "Have the person...do this:"
Maybe the step is really 

"4. Alert the recipient that the deployment .jar is available so that they can:
      a. Access the .jar file.
      b. Change environments to that in which....   [Does this mean they have to move the .jar there?]
      c. Click....

Reply:
The way it works is that from RM they follow the steps for deploying. They will be given a selection that allows them to upload the .jar file. So, we don't want to rewrite the RM deploy steps - just give them the info on the jar file.

Reply:
Fixed.